cmd/clair/httptransport.go

```diff
@@ -60,7 +61,7 @@ func devMode(ctx context.Context, conf config.Config) (*http.Server, error) {
 	matcher.Register(mux)
 	return &http.Server{
 		Addr:    conf.HTTPListenAddr,
-		Handler: mux,
+		Handler: Compress(mux),
```

Comment:
@hdonnay 

Looks good. One question, do we want this to be configurable ? I.e. do not chain the middleware if a flag is not present ?

Reply:
I don't think so, my thinking that it's client-side opt-in already.